tests/Commands/HoneybadgerCheckinCommandTest.php

```diff
@@ -8,9 +8,16 @@
 use Honeybadger\HoneybadgerLaravel\Commands\HoneybadgerCheckinCommand;
 use Honeybadger\Tests\TestCase;
 use Illuminate\Contracts\Console\Kernel;
+use Illuminate\Support\Facades\Config;
 
 class HoneybadgerCheckinCommandTest extends TestCase
 {
+    protected function setUp(): void
```

Comment:
This fixes the failing tests (they are also failing on master).

Reply:
Ah, thanks